tracking/src/Tracker.js

```diff
@@ -102,15 +102,16 @@ class Tracker {
     sendEvent(action, dataObj, callback) {
         const path = `${this.baseUrl}/__va.gif`; 
         let defaultsParams = this.dataCollection.toJson(); 
-        defaultsParams.uid = this.sessionManager.getUserID();
         defaultsParams.code = this.code; 
 
-        let params = Object.assign(defaultsParams, dataObj); 
+        let params = Object.assign(dataObj, defaultsParams); 
         params.sid = this.sessionManager.getSessionId(); 
+        params.uid = this.sessionManager.getUID(); 
         params.sdk = SDK; 
         params.v = SDK_VERSION;     
         params.action = action; 
         params.ts = new Date().getTime(); 
```

Comment:
can we refactor default params into dataCollections?
it will be cleaner to consolidate all non-user editable params into 1 place.

Example:
```
        let defaultsParams = this.dataCollection.toJson(action); 
```

or

```
        let defaultsParams = this.dataCollection.toJson(action, this.sessionManager.getSessionId(), this.sessionManager.getUID()); 
```

Reply:
changed already